app/views/posts/index.html.erb

```diff
@@ -1,10 +1,47 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-  </head>
-
-  <body>
-    <h1>Here is a list of posts for a given user.</h1>
-  </body>
-</html>
+<div class="posts container">
+
+  <div class="user_item row">
+    <div class="user_img">
+      <img src=<%= @user.photo %> alt="user_img">
+    </div>
+
+    <div class="user_desc_cont">
+      <a href=<%= user_url(@user.id) %>>
+        <div class="user_desc col">
+          <h2 class="user_name"><%= @user.name %></h2>
+          <p class="user_post_number">Number of posts: <%= @user.posts_count %></p>  
+        </div>
+      </a>
+    </div> 
+  </div>
+
+  <ul class="post_list post_list_show col">
+    <% @posts.each do |post| %>
+      <li class="post_item_desc">
+
+        <a href=<%= user_post_url(user_id: @user.id, id: post.id) %>>
+          <div class="post_item">
+            <h4 class="post_title"><%= post.title %></h4>
+            <p class="post_text"><%= post.text %></p>
+            <p class="comments_likes">
+              Comments: <%= post.comments_count %>, 
+              Likes: <%= post.likes_count %>
+            </p>
+          </div>
+        </a>
+
+        <div class="post_comments">
+          <% post.recent_comments.each do |comment| %>
```

Comment:
- [ OPTIONAL ] I think it would be great if you just display the most three recent posts as mentioned in the requirements since you created this method already so it is time to use it and you can just display the most three recent posts and when the user press on display all posts so he can see all the posts 💯👍🏻🔥💪🏻 and I am assigin it as optional because the previous reviewer didn't mention it but it still important to implement this requirement 🚀⭐

![image](https://github.com/Lawmsangi/rails-blog-app/assets/58553711/1d8894d8-4278-4ebf-aae0-2af7ee97d757)

- The given wireframe ✨
![image](https://github.com/Lawmsangi/rails-blog-app/assets/58553711/8d5f0938-8c6e-4705-ad18-ccf5d574f0a2)


![image](https://github.com/binyamolango/Blog-App/assets/58553711/3f2b23c0-36e5-4454-9d90-6e62807f1a81)

Reply:
@Reem-lab I will implement it. Thank you.